### ISSUE TITLE
[llamacpp] - <title>Llama.CPP HTTP server. 

### ISSUE THREAD

**Issue by haoyu-zhao** [OpenBMB/MiniCPM-o#457]
### 摘要 | Summary

Hi, I am trying to load this using Llama.CPP HTTP server. However, it failed to return the correct answer when called. Is this method not supported now?
![image](https://github.com/user-attachments/assets/6f179666-67d4-47cd-9261-211f3a071be9)

### 基本示例 | Basic Example

llama.cpp api server

### 缺陷 | Drawbacks

No Drawbacks

**Comment by tc-mb** [comment]
Yes, I haven't taken the time to adapt this part of llamacpp yet.

**Comment by haoyu-zhao** [comment]
If possible, I would greatly appreciate it if you could prioritize fixing this issue at your earliest convenience. This feature is particularly important for my current work, and having it operational would significantly aid in my progress. Thank you very much for your understanding and assistance!
